### ISSUE TITLE
Add glob support to Incremental CLI argument `--incremental=glob`

### ISSUE THREAD

**Issue by zachleat** [11ty/eleventy#3335]
`--incremental=filename` shipped in #3324 but it’d be nice to have `--incremental=glob` too.

**Comment by zachleat** [comment]
Probably dependent on #3334

**Comment by zachleat** [comment]
> By default this repo closes enhancement request issues and places them in a queue for folks to upvote. We do this to avoid a large build-up of open issues. We got this idea from the [lodash project](https://twitter.com/samselikoff/status/991395669016436736).

> Don’t forget to upvote the first comment with 👍 to register your vote! You can view [the current enhancement backlog](https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+).
